cimr/processor/utils.py

```diff
@@ -12,6 +12,25 @@
 import logging
 import subprocess
 
+def set_chrom_dict():
+    """Make a dictionary to standardize chromosome IDs in input files."""
+    maxchrom = 23
+    chrom_dict = {str(i):'chr' + str(i) for i in range(1, maxchrom)}
+    chrom_dict.update({'chr'+str(i):'chr' + str(i) for i in range(1, maxchrom)})
```

Comment:
Um why is this step needed? I get you have a dictionary [numbers->chromosomes], but why is the reverse needed?

Reply:
Great catch! I fixed it. Thank you!